steampipe.spvars.example

```diff
@@ -0,0 +1,5 @@
+# List of trusted repository admins
+trusted_repo_admin = ["abc20", "abc21"]
+
+# List of trusted issue admins
+trusted_issue_admin = ["abc20", "abc21"]
```

Comment:
```suggestion
trusted_issue_admins = ["abc20", "abc21"]
```
In the AWS perimeter mod, we have used plural form for the variables and I believe we should continue the same format here as well - https://github.com/turbot/steampipe-mod-aws-perimeter/blob/main/steampipe.spvars.example